collector/topological_inventory.py

```diff
@@ -54,6 +54,28 @@ def _load_yaml(filename: str) -> dict:
 APP_CONFIG = _load_yaml(f'{CFG_DIR}/topological_app_config.yml').get(APP_NAME)
 QUERIES = _load_yaml(f'{CFG_DIR}/topological_queries.yml')
 
+Tenant = namedtuple('Tenant', ('account_number', 'headers'))
```

Comment:
Use of `namedtuple` is interesting!

Technically, `Tenant` could be a dictionary too, right?

Reply:
Yes, of course, it can be a dict with no issues. There was no problems with that. It was just this:

```python
tenant.headers
# vs
tenant['headers']
```

The later didn't fit in 80 char line somewhere for me, so I start playing with Python toys. :smile: 

Obviously, there's no real need to use the `namedtuple` over a dict. Maybe a tiny bit of memory, but that's so marginal, I'm ashamed to even mention it. If you don't like it or would feel better with a regular dict, it can be easily removed. :slightly_smiling_face: It's just... a little fun. :wink:

Reply:
I learned something new and a little fun is always good!
- So fine by me! :)